apps/web/app/future/settings/(settings)/developer/api-keys/page.tsx

```diff
@@ -16,6 +17,11 @@ export const generateMetadata = async () =>
     (t) => t("create_first_api_key_description", { appName: APP_NAME })
   );
 
+async function revalidateApiKeys() {
+  "use server";
+  revalidatePath("/future/settings/developer/api-keys");
+}
+
```

Comment:
Ideally this would live in a client component and we can just call it but since we need the child component also in pages we can't do that yett